src/temp/Explore/ExploreButton.tsx

```diff
@@ -0,0 +1,149 @@
+        </div>
+        <div className={style.switchContainer}>{switchComponent}</div>
+      </div>
+    </>
+  );
+};
+
+export const YearRangeSlider = ({
+  startYear,
+  endYear,
+}: YearRangeSliderProps) => {
+  const minDistance = 10;
+  const [value1, setValue1] = useState<number[]>([20, 37]);
+
+  function valuetext(value: number) {
+    return `${value}°C`;
+  }
+
+  const handleChange1 = (
+    event: Event,
+    newValue: number | number[],
+    activeThumb: number
+  ) => {
+    if (!Array.isArray(newValue)) {
+      return;
+    }
+
+    if (activeThumb === 0) {
+      setValue1([Math.min(newValue[0], value1[1] - minDistance), value1[1]]);
+    } else {
+      setValue1([value1[0], Math.max(newValue[1], value1[0] + minDistance)]);
+    }
+  };
+
+  return (
+    <div className={style.rangeMainContainer}>
+      <div className={style.rangeContainer}>
+        <div className={style.playIconContainer}>
+          <PlayIcon />
+        </div>
+        <div className={style.sliderContainer}>
+          <SmallSlider
+            getAriaLabel={() => 'Minimum distance'}
+            value={value1}
+            onChange={handleChange1}
+            getAriaValueText={valuetext}
+            disableSwap
+          />
+        </div>
+      </div>
+      <div className={style.startYear}>{startYear}</div>
+      <div className={style.endYear}>{endYear}</div>
+    </div>
+  );
+};
+
+const ExploreButton = ({
+  label,
+  isOpen,
+  startYear,
+  endYear,
+}: ExploreButtonProps) => {
+  const { t } = useTranslation(['allProjects']);
+
+  return (
+    <>
+      <CustomButton startIcon={<ExploreIcon />}>{label}</CustomButton>
+
+      {isOpen ? (
+        <div className={style.exploreMainContainer}>
+          <div className={style.exploreContainer}>
+            <div>
+              <EcosystemOption
+                infoIcon={<InfoIcon />}
+                label={t('allProjects:currentForests')}
+                switchComponent={<StyledSwitch currentForestSwitch />}
+              />
+              <div className={style.hrLine} />
+              <EcosystemOption
+                infoIcon={<InfoIcon />}
+                label={t('allProjects:restorationPotential')}
+                switchComponent={<StyledSwitch restorationSwitch />}
+              />
+              <div className={style.hrLine} />
+              <EcosystemOption
+                infoIcon={<InfoIcon />}
+                label={t('allProjects:deforestation')}
+                switchComponent={<StyledSwitch deforestrationSwitch />}
+              />
+              <YearRangeSlider startYear={startYear} endYear={endYear} />
+              <div className={style.hrLine} />
+              <EcosystemOption
+                infoIcon={undefined}
+                label={t('allProjects:projects')}
+                switchComponent={<StyledSwitch />}
+              />
+            </div>
+            <div className={style.exploreDescription}>
+              {t('allProjects:exploreDescription')}
```

Comment:
This already has a translation presented as `3trilliontrees` in `maps.json`. Also, since we don't want repetitive translations, kindly check before adding a new translation.